website/docs/feature-flag-tutorials/use-cases/trunk-based-development.md

```diff
@@ -0,0 +1,222 @@
+
+-   The feature or functionality the flag is associated with
+-   The environment or user segment the flag is intended for (e.g., "prod", "beta", "mobile\*users")
+-   A version or iteration number, if applicable
+-   A short, descriptive prefix (e.g., "ff\*", "feat\_"). For example: `ff_checkout_flow_v2_prod` or `feat_new_dashboard_beta`.
+
+#### Keep the Trunk Deployable
+
+The trunk should always be deployable, enabling teams to accelerate release cycles and respond quickly to market demands.
+To keep the trunk in a deployable state, keep the flag off in your production environment. The code for your feature should be wrapped in your flag and, therefore, not executable in production, even though the code will be deployed frequently. This ensures users will not have access to the feature before its official release. Additionally, this promotes collaboration by allowing teams to work independently on different features without interference.
+
+To verify the functionality of your feature while it is incomplete, enable the flag in the development environment in the Unleash Admin UI. In some cases, you may also find it valuable to enable the flag in a testing/QA environment. Unleash environment-specific flag configurations make it easy to manage these different states across your [environments](/reference/environments#how-to-start-using-environments). You can quickly toggle flags on or off for specific environments, ensuring the trunk remains deployable in production while enabling active development and testing in other contexts. Use the default production environment toggle in Unleash to enable your flag when you’re ready to make your feature available.
+
+:::note
+Depending on the size and scope of a feature you’re developing, you may need more than one flag. Generally, we recommend creating as few flags as possible per feature, as making too many flags associated with one feature can become more complex to manage over time with trunk-based development. Our documentation on [best practices for feature flags at scale](/topics/feature-flags/best-practices-using-feature-flags-at-scale) provides more concrete details on large-scale feature flag management.
+:::
+
+#### Leverage Tagging and Metadata
+
+In addition to meaningful names, apply relevant tags and metadata to your feature flags in Unleash. This metadata can include information such as:
+
+-   The team or product area responsible for the flag
+-   The release timeline or planned retirement date
+-   Associated Jira tickets or GitHub issues
+-   Relevant documentation or context
+-   [Tagging](/reference/tags) and metadata make it easier to search, filter, and manage your feature flags, especially as the number of flags grows.
+
+Click **Create new tag** in the Unleash Admin UI to add these details to your feature flag.
```

Comment:
I think we need a phrase to connect this section with the previous. Also, it's not clear where in the Unleash Admin UI we are

Reply:
The sections were switched around. I updated this.